### DISCUSSION TITLE
Edit (change) Aspect Type

### DISCUSSION THREAD

**Discussion by GerlofdeHaan** [alkem-io/alkemio#593] [Ideas] [answered]
At this moment it is not possible to edit (change) the Aspect Type after you created a new Aspect.

**Comment by techsmyth** [comment, marked as answer]
Hi Gerlof,

This should be available near term, it is pretty much at the top of the backlog and in the current sprint: https://github.com/alkem-io/server/issues/1880

Cheers, Neil
